whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java

```diff
@@ -55,6 +56,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(MessageParser.class);
 
     private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("EEE MMM dd HH:mm:ss z yyyy");
+    public static final String HEADER_BASE_64 = "Content-Transfer-Encoding: base64";
```

Comment:
This is both the header and the value, maybe define each separately (also it's not guaranteed there will only be one space separating them).

Reply:
Changed it to use pattern matching.

Reply:
👍